src/store/reducers/settings.js

```diff
@@ -16,6 +23,9 @@ const savedQueries = window.web_version
 const savedTenantGeneralTab = window.web_version
     ? userSettings[TENANT_INITIAL_TAB_KEY]
     : getValueFromLS(TENANT_INITIAL_TAB_KEY);
+export const queryInitialRunAction = window.web_version
```

Comment:
Here is a lot of initial settings by now. Maybe it would be useful if we create a helper function that retrieves setting value depends on `window.web_version`. And in `initialState` values will be this helper calls without many constants above?